api/src/test/scala-2.12/scalatex/Scala212ErrorTests.scala

```diff
@@ -0,0 +1,56 @@
+package scalatex
+
+import utest._
+import scalatex.stages._
+import scalatags.Text.all._
+import scalatex.Internals.{DebugFailure, twRuntimeErrors, twfRuntimeErrors}
+import ErrorTests.check
+
+object Scala212ErrorTests extends TestSuite {
+  val tests = TestSuite {
+    'chained {
+      'curlies {
+        * - check(
+          twRuntimeErrors("@p{@Seq(1, 2, 3).foldLeft(0)}"),
+          "type mismatch",
```

Comment:
I have some concerns about this test. On scala 2.11 it throws "missing argument list for method foldLeft", as expected, but on scala 2.12 it complains about type mismatch. If remove `@p{...}` it produce a correct error (about missing arguments) and `@p{@Seq(1, 2, 3).foldLeft(0)(_ + _)}` throws no error, so I think the behaviour is correct, but anyway it worth to give it some attention.